x-pack/plugins/security_solution_serverless/server/cloud_security/constants.ts

```diff
@@ -40,7 +40,7 @@ export const METERING_CONFIGS = {
   },
   [CLOUD_DEFEND]: {
     index: CLOUD_DEFEND_HEARTBEAT_INDEX,
-    assets_identifier: 'agent.id',
+    assets_identifier: 'agent.id.keyword',
```

Comment:
So @CohenIdo are we going to fix mappings of d4c integration first? so we will use `agent.id` instead?